### PR TITLE
[MINOR] Fixes for py27/py35 compatibility

### PR DESCRIPTION
**Pull request by abacigalup-eb**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+services:
+    tox:
+        build:
+            context: .
+        environment:
+            - PIP_INDEX_URL=https://aws:${CODEARTIFACT_AUTH_TOKEN}@eventbrite-shared-353605023268.d.codeartifact.us-east-1.amazonaws.com/pypi/python/simple/
```
Comment (gastudillo-eb) [inline docker-compose.yml:6]: nl

File: Dockerfile [hunk 2]
```diff
@@ -1,11 +1,7 @@
...
 
 COPY . .
 
-# In case an internal dependency is added, we will need to look in our internal codearficato repo.
-# But as that is currently not true, we can look for them in pypy making things simpler. 
-RUN pip install -e .[tests]
-
-CMD ["/bin/bash", "-c", "py.test && coverage xml -o /tmp/codeartifact"]
\ No newline at end of file
+CMD tox
```
Comment (gastudillo-eb) [inline Dockerfile:7]: nl

File: tox.ini [hunk 3]
```diff
@@ -0,0 +1,21 @@
+[tox]
+envlist =
+    py{27,35}-django18
+
+[testenv]
+deps =
+    # Django versions
+    django18: django==1.8.19
```
Comment (gabrielf-eb) [inline tox.ini:8]: no need django, right?

File: docker-compose.yml (hunk 1 above)
Comment (abacigalup-eb) [reply docker-compose.yml]: np

File: Dockerfile (hunk 2 above)
Comment (abacigalup-eb) [reply Dockerfile]: np

**Review by gabrielf-eb — APPROVED** [review]

**Review by ghidalgo-eb — COMMENTED** [review]
https://github.com/eventbrite/pilo/blob/main/.circleci/config.yml#L62
This line will probably need to be updated so the library is available in python3 in our wheel repositories
changing to py37 should work fine

**Review by ghidalgo-eb — APPROVED** [review]

**Review by gastudillo-eb — APPROVED** [review]

**Pull request merged**